TSC-Charter.md

```diff
@@ -81,21 +81,31 @@ Subject to such policies as may be set by the Board, the TSC is
 responsible for all technical development within the Node.js Foundation,
 including:
 
-* Setting release dates.
-* Release quality standards.
-* Technical direction.
-* Project governance and process (including this policy).
-* GitHub repository hosting.
-* Conduct guidelines.
-* Maintaining the list of additional Collaborators.
-* Development process and any coding standards.
+* Setting release dates
+* Release quality standards
+* Technical direction
+* Maintaining the list of additional Collaborators
+* Development process and any coding standards
 * Mediating technical conflicts between Collaborators or Foundation
-projects.
+projects
+* Management of the `nodejs-private` GitHub organization
 
 The TSC will define Node.js Foundation’s release vehicles and serve as
 Node.js Foundation’s primary technical liaison body with external open
 source projects, consortiums and groups.
 
+The TSC shares responsibility with the Node.js Community Committee ("CommComm")
```

Comment:
nit: Add link to responsibilities on both side.

Replace:
Node.js Community Committee ("CommComm") for"

With

[Node.js Community Committee ("CommComm") Charter](https://github.com/nodejs/community-committee/blob/master/Community-Committee-Charter.md#section-5-responsibilities-of-the-commcomm) for: